src/storage/v2/delta.hpp

```diff
@@ -123,6 +124,7 @@ inline bool operator!=(const PreviousPtr::Pointer &a, const PreviousPtr::Pointer
 struct Delta {
   enum class Action {
     // Used for both Vertex and Edge
+    DELETE_DESERIALIZED_OBJECT,
```

Comment:
As this is only used for disk, I would make a comment which should explain usage od delta action enums, which are used for in memory and which for disk storage